resources/dicts/relationship_events/normal_interactions/trust/increase.json

```diff
@@ -1,349 +1,400 @@
+			"m_c and r_c chase off a predator on a patrol together.",
+			"m_c and r_c make an elaborate plan to evict a predator from the Clan's territory.",
+			"m_c and r_c volunteer to go on an extra border patrol together.",
+			"m_c notices that r_c has finished all {PRONOUN/r_c/poss} tasks without issues lately.",
+			"During a patrol, m_c and r_c have a long conversation about their beliefs and agree on many things.",
+			"During the last gathering, m_c and r_c defended their Clan against another Clan's heckling."
+		],
+		"main_status_constraint": [
+			"apprentice",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"medicine cat apprentice",
+			"leader"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader",
+			"medicine cat apprentice"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c a deep, dark secret.",
+			"m_c confesses something to r_c.",
+			"m_c depends on r_c to back {PRONOUN/m_c/object} up during a debate.",
+			"m_c and r_c swap deep, dark secrets with each other."
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_high4",
+		"intensity": "high",
+		"interactions": [
+			"m_c goes to r_c for advice about some recent troubles.",
+			"m_c asks r_c to help {PRONOUN/m_c/object} find the right words to say to a Clanmate.",
+			"m_c asks r_c for help resolving an argument with a friend."
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"random_status_constraint": [
+			"elder",
+			"mediator",
+			"mediator apprentice"
+		]
+	},
+	{
+		"id": "trust_star_inc_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c about {PRONOUN/m_c/poss} last encounter with StarClan.",
+			"m_c debates about the meaning of a prophetic dream with r_c.",
+			"r_c puts a lot of faith into m_c's skill as {PRONOUN/m_c/poss} rank."
+		],
+		"main_status_constraint": [
+			"medicine cat apprentice",
+			"medicine cat",
+			"leader"
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"random_status_constraint": [
+			"elder",
+			"warrior",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_led_inc_high1",
+		"interactions": [
+			"m_c talks with r_c about difficult decisions.",
+			"m_c makes battle plans with r_c.",
+			"m_c discusses future plans for the Clan with r_c.",
+			"m_c catches up on the current Clan relationships with r_c.",
+			"m_c discusses plans to expand the territory with r_c."
+		],
+		"relationship_constraint": [
+			"trust_50"
+		],
+		"main_status_constraint": [
+			"leader"
+		],
+		"random_status_constraint": [
+			"deputy",
+			"elder",

```

Comment:
it's important to remember for a lot of interactions throughout this PR that elders can be as young as 6 moons old. personally for social inclusions of elders, i would recommend making it an age constraint instead, though admittedly i'm not too sure how that would work in tandem with status constraints. mayhaps worth considering sections for cats who are both seniors and elders

Reply:
I forgot that elders can be so young!! The original file just had the line:
"m_c talks with r_c about difficult decisions."
so I just wanted to spruce it up a little. I think I'll just remove the ones I added and put them into a new section for leaders, medicine cats, and deputies.